### PR TITLE
Add more install and getting started material

### PR DESCRIPTION
**Pull request by tusmester**
- [x] Separate install guide for Services nuget package.
- [ ] new doc: about our components and how to choose from them (do I need this? do I need that?)

### CODE REVIEW

**Comment by borsi** [comment]
Good thinking. This doc could be a great staring point for every developer who'd want to know what she can do with sensenet in the first place. Where should it go in your opinion? 

It could be part of the getting started guide as well, right?

**Comment by tusmester** [comment]
You mean the main readme? Because that is the only getting started guide we have right now.

Yes, this could be a section of the main readme; I was just thinking of a separate doc so that it can be linked/displayed on the github.io page (main readme files are not linked there, only the ones under the doc folder).

**Comment by borsi** [comment]
No, I was thinking that we could have separate getting started guides of which this - also separate - document could be part of. 🐙

File: docs/install-sn-from-nuget.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Install sensenet ECM 7.0 from NuGet
-This article is **for developers** about installing the core layer, **sensenet Services**, but before getting into that, please take a look at the concept of our install method.
+This article is **for developers** about installing the core layer, [sensenet Services](https://github.com/SenseNet), but before getting into that, please take a look at the concept of our install method.
```
Comment (borsi) [inline docs/install-sn-from-nuget.md]: shouldn't this url be https://github.com/SenseNet/sensenet?

Comment (tusmester) [inline docs/install-sn-from-nuget.md]: Thanks, fixed it.

**Pull request merged**